src/lib/arch/aarch64/ExceptionHandler.cc

```diff
@@ -557,6 +557,7 @@ bool ExceptionHandler::init() {
         size_t length = registerFileSet.get(R1).get<size_t>();
 
         int64_t result = linux_->munmap(addr, length);
+        result = result >= 0 ? 0 : result;
```

Comment:
done.

Reply:
incase the diff doesn't work here is what I've written:
```
        // if successful, sysHandler_->munmap returns the total number of bytes
        // unmapped. if the value is greater than 0, 0 is returned as specified
        // by the actual munmap specification. However, all negative values
        // returned by munmap are in accordance with the munmap specification,
        // so in case a negative value is returned it will remain the same.
```

Reply:
If on first and second line needs to be a capital

Reply:
If on second line is still lower case

Reply:
Sorry. corrected now